apps/user-office-frontend/src/components/proposal/ProposalCreate.tsx

```diff
@@ -54,8 +54,8 @@ export default function ProposalCreate() {
     callId: string;
     templateId: string;
   }>();
-  const { questionarySteps } = useBlankQuestionaryStepsData(
-    parseInt(templateId as string)
+  const { questionarySteps } = useBlankQuestionaryStepsDataByCallId(
```

Comment:
question: What is the need for changing useBlankQuestionaryStepsData to useBlankQuestionaryStepsDataByCallId ?